.github/workflows/MinioTests.yml

```
@@ -0,0 +1,66 @@
+name: Minio Tests
+on: [push, pull_request,repository_dispatch]
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.head_ref || '' }}-${{ github.base_ref || '' }}-${{ github.ref != 'refs/heads/main' || github.sha }}
+  cancel-in-progress: true
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  minio-tests:
+    name: Minio Tests
+    runs-on: ubuntu-20.04
```

```suggestion
    runs-on: ubuntu-24.04
```
(ubuntu-20 will go away, either 22 or 24)